uber/models.py

```diff
@@ -1025,6 +1035,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     placeholder   = Column(Boolean, default=False, admin_only=True)
     first_name    = Column(UnicodeText)
     last_name     = Column(UnicodeText)
+    legal_name_   = Column('legal_name', UnicodeText, nullable=True, default=None)
```

Comment:
Two comments about this:

1) Our convention in this codebase for optional ``UnicodeText`` columns is to simply say ``Column(UnicodeText)`` which requires them to *not* be NULL but which also sets a default value of the empty string if they're unset.  This makes it much easier when writing code which references such columns, because you know you're dealing with a string (albeit possibly an empty one) and thus don't have to sprinkle checks for ``None`` checks throughout your code.  I think we should change this to simply use the same convention as e.g. ``first_name`` and ``last_name`` in order to disallow NULL values and use the empty string in that case.

2) We currently have a bug in our Tracking code which stops us from being able to store records when the column name is different than the property name (#2164).  In this case it'll probably happen to work anyway, since we do have a ``legal_name`` property which will do the correct thing, but we need to make sure to test the Tracking table before this is merged, since it would be Really Really Bad if attendee objects stopped getting tracked because of that bug.